x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/normalization/rule_converters.ts

```diff
@@ -464,6 +473,15 @@ export const convertCreateAPIToInternalSchema = (
 ): InternalRuleCreate => {
   const typeSpecificParams = typeSpecificSnakeToCamel(input);
   const newRuleId = input.rule_id ?? uuidv4();
+
+  const throttleAndNotifyWhen =
+    !input.actions?.length || !input.actions[0].frequency
```

Comment:
nit: And it's the same as above (`!actions.length || !actions[0].frequency`), it's worth considering moving to a separate function.